modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/RocksDbMvPartitionStorage.java

```diff
@@ -559,8 +575,21 @@ public void addWriteCommitted(RowId rowId, @Nullable BinaryRow row, HybridTimest
             // We only write tombstone if the previous value for the same row id was not a tombstone.
             // So there won't be consecutive tombstones for the same row id.
             if (!newAndPrevTombstones) {
+                //TODO IGNITE-16913 Add proper way to write row bytes into array without allocations.
+                ByteBuffer rowBytes;
+
+                if (row == null) {
+                    rowBytes = allocateDirect(0);
```

Comment:
Hm, this just looks wrong, allocating a buffer with size 0. Maybe it could be a constant